Source/Router.swift

```diff
@@ -101,6 +102,7 @@ public final class Router {
     }
     
     private var routes: [String : Route] = [:]
+    private var canceledRequests: [NSURLProtocol] = []
```

Comment:
I solved the retain cycle by remembering the `NSURL`, which should be canceled.
fixed in c10b110fa7b1f30d1a5c9ad6a97e81917c99958f